### DISCUSSION TITLE
Xcode 16 Required

### DISCUSSION THREAD

**Discussion by mikaelacaron** [mikaelacaron/Basic-Car-Maintenance#318] [Announcements]
Now requiring Xcode 16 to run this project!

The main reason is because I've converted the project to use **Folders** as opposed to **Groups**. Which I hope will cut down on merge conflicts when creating new files.

Files / Folders are organized in alphabetical order in the navigator, and you can't move them around.

See more differences in this blog post: https://troz.net/post/2024/xcode_folders_groups/

**Comment by horacio3m** [comment]
Is there any plan for Hacktober this year?

**Reply by mikaelacaron** [reply]
Yup! @horaciomuller it's already started! like 10 issues have already been assigned to people in the last 4 days

See this post!
https://github.com/mikaelacaron/Basic-Car-Maintenance/discussions/303
